tests/components/min_max/test_sensor.py

```diff
@@ -58,6 +60,7 @@ def test_min_sensor(self):
         assert self.max == state.attributes.get("max_value")
         assert entity_ids[1] == state.attributes.get("max_entity_id")
         assert self.mean == state.attributes.get("mean")
+        assert self.median == state.attributes.get("median")
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5872960)